app/domain/authentication/authn_k8s/validate_pod_request.rb

```diff
@@ -13,16 +13,17 @@ module AuthnK8s
         validate_webservice_is_whitelisted:  ::Authentication::Security::ValidateWebserviceIsWhitelisted.new,
         validate_role_can_access_webservice: ::Authentication::Security::ValidateRoleCanAccessWebservice.new,
         enabled_authenticators:              Authentication::InstalledAuthenticators.enabled_authenticators_str(ENV),
-        validate_resource_restrictions:      ValidateResourceRestrictions.new,
-        extract_container_name: ExtractContainerName.new,
-        logger: Rails.logger
+        validate_resource_restrictions:      Authentication::ResourceRestrictions::ValidateResourceRestrictions.new(
```

Comment:
you can remove the `Authentication` module from these deps as well.

Reply:
done